src/pg/sql/select.js

```diff
@@ -1,47 +1,30 @@
 import sql, { join, raw, empty } from 'sql-template-tag';
 import getArgSql from './getArgSql.js';
-import getIdMeta from './getIdMeta.js';
-import getSelectCols from './getSelectCols.js';
-// import debug from 'debug';
-// const log = debug('graffy:pg:select');
+import { getIdMeta } from './getMeta.js';
+import { getSelectCols } from './helper.js';
 
 const MAX_LIMIT = 4096;
 
-export function selectByArgs(args, options, { forUpdate = false } = {}) {
+export function selectByArgs(args, options) {
   const { table } = options;
   const { where, order, limit, meta } = getArgSql(args, options);
-  const clampedLimit = forUpdate
-    ? 1
-    : Math.min(MAX_LIMIT, Math.max(0, limit || 0));
-
+  const clampedLimit = limit || MAX_LIMIT;
```

Comment:
We need `Math.min(MAX_LIMIT, Math.max(0, limit || 0));` here, to clamp limits that are higher than MAX_LIMIT.